substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/methodhandles/Target_java_lang_invoke_MethodHandleNatives.java

```diff
@@ -219,6 +219,9 @@ static Target_java_lang_invoke_MemberName resolve(Target_java_lang_invoke_Member
         }
         return resolved;
     }
+
+    @Delete
+    static native MethodHandle linkMethodHandleConstant(Class<?> callerClass, int refKind, Class<?> defc, String name, Object type);
```

Comment:
This and https://github.com/graalvm/graalvm-community-jdk21u/pull/35/files#diff-9340f27b20355e6b83f09acff296c1f69c4ec8bb45d9900519ae9ba51bd4e2b1R89 are not necessary to fix the issue at hand, right? You are just preemptively backporting them.